src/_kw

```diff
@@ -455,7 +455,10 @@ _kw_ssh()
   _arguments : \
     '(-r --remote)'{-r,--remote}'[explicitly tell which target machine to ssh into]: : ' \
     '(-s --script -c --command)'{-s,--script}'[execute a bash command in the target machine]: : ' \
-    '(-c --command -s --script)'{-c,--command}'[execute a bash script in the target machine]: : '
+    '(-c --command -s --script)'{-c,--command}'[execute a bash script in the target machine]: : ' \
+    '--send[send a file or directory to the remote machine]: : ' \
+    '--get[get a file or directory from the remote machine]: : ' \
+    '--to[specify the destination path]: : '
```

Comment:
@davidbtadokoro could you review this part?

Reply:
Sorry for the delay, I missed this ping 😥
 
Below is the diff for the implementation of (what I think is) the correct completions. I made the following assumptions:
1) `--get`, `--send`, `--command`, and `--script` are mutually exclusive (can't be used in the same command), so by using one of those, the others should not be suggested for completion.
2) `--to` should only be suggested after using `--get` or `--send`.
3) `--remote` can be suggested with both `--get` and `--send` (and with all options, as a matter of fact).

Please @aquilamacedo, verify if these assumptions are correct.

```
diff --git a/src/_kw b/src/_kw
index 0a02734..dfd2ad4 100644
--- a/src/_kw
+++ b/src/_kw
@@ -452,13 +452,19 @@ _kw_self-update()
 _kw_s() { _kw_ssh }
 _kw_ssh()
 {
+  local to=''
+
+  if ((words[(I)--send|--get])); then
+    to='--to[specify the destination path]: : '
+  fi
+
   _arguments : \
     '(-r --remote)'{-r,--remote}'[explicitly tell which target machine to ssh into]: : ' \
-    '(-s --script -c --command)'{-s,--script}'[execute a bash command in the target machine]: : ' \
-    '(-c --command -s --script)'{-c,--command}'[execute a bash script in the target machine]: : ' \
-    '--send[send a file or directory to the remote machine]: : ' \
-    '--get[get a file or directory from the remote machine]: : ' \
-    '--to[specify the destination path]: : '
+    '(-s --script -c --command --send --get)'{-s,--script}'[execute a bash command in the target machine]: : ' \
+    '(-c --command -s --script --send --get)'{-c,--command}'[execute a bash script in the target machine]: : ' \
+    '(-s --script -c --command --get)--send[send a file or directory to the remote machine]: : ' \
+    '(-s --script -c --command --send)--get[get a file or directory from the remote machine]: : ' \
+    $to
 }

 _kw_vars()
```

Reply:
Thx @davidbtadokoro. Your resolution is excellent. I hadn't noticed these details before